dataprep/eda/create_report/__init__.py

```diff
@@ -0,0 +1,46 @@
+"""
+    This module implements the create_report(df) function.
+"""
+
+from typing import Optional
+from pathlib import Path
+import pandas as pd
+from bokeh.resources import CDN
+from jinja2 import Environment, PackageLoader, select_autoescape
+from .formatter import format_report
+
+__all__ = ["create_report"]
+
+ENV_LOADER = Environment(
+    loader=PackageLoader("dataprep", "eda/create_report/templates"),
+)
+
+
+def create_report(
+    df: pd.DataFrame,
+    title: Optional[str] = "DataPrep Report",
+    mode: Optional[str] = "basic",
+) -> None:
+    """
+    This function is to generate and render element in a report object.
+
+    Parameters
+    ----------
+    df
+        The DataFrame for which data are calculated.
+    title
+        The title of the report.
+    mode
+        There are 3 modes which user can use.
+        basic: Genreate a pandas-profiling-like report.
+        full: Generate more detailed version than basic.
+        mini: Generate a brief conclusion of the report.
```

Comment:
We probably shouldn't have these options in the docs if they're not implemented.